modules/AOP_Case_Updates/AOP_Case_Updates.php

```diff
@@ -271,7 +285,9 @@ public function sendEmail(
         $signature = array(),
         $caseId = null,
         $addDelimiter = true,
-        $contactId = null
+        $contactId = null,
+        $txtDelimiter = "",
+        $lang = "default"
```

Comment:
Same as before, changing the signature of this method is most likely going to break things.